Sources/Navidux/Core/Navigation/NavigationAction/Actions/PopRoutingAction.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PopNavigationAction.swift
+//
+//
+//  Created by Tamerlan Satualdypov on 13.02.2024.
+//
+
+import UIKit
+
+public struct PopNavigationAction: NavigationAction {
+    private let screenClass: UIViewController.Type?
+    
+    init(screenClass: UIViewController.Type?) {
+        self.screenClass = screenClass
+    }
+    
+    public func perform(on coordinator: Coordinator) {
+        guard
+            let screenClass = self.screenClass,
+            let screen = coordinator.navigationController.screens.last(where: { $0.isKind(of: screenClass) })
+        else {
+            coordinator.navigationController.popViewController(animated: true)
```

Comment:
I see your point, we should allow user control whether action needs animation or not, it should not passed as an argument in the function, because this function is in `NavigationAction` protocol and not every action needs this argument. I will look for better solution